### PR TITLE
Shortcuts Overlay

### PR DESCRIPTION
**Pull request by pshriwise**
Adds a keyboard shortcuts overlay so users don't have to parse through the README to figure these things out. Should display the correct shortcuts for Ubuntu or OSX.

### CODE REVIEW

File: overlays.py [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+            for j in range(n_cols):
+                self.tableWidget.setItem(i, j, ShortcutTableItem())
+
+        self.tableWidget.setShowGrid(False)
+        self.tableWidget.setSizePolicy(QSizePolicy.Expanding,
+                                       QSizePolicy.Expanding)
+        self.tableWidget.verticalHeader().setVisible(False)
+        self.tableWidget.horizontalHeader().setVisible(False)
+        self.tableWidget.setStyleSheet("background-color:"
+                                       "rgba(30, 30, 30, 230);"
```
Comment (paulromano) [inline overlays.py]: Combine this with previous line

File: overlays.py [hunk 2]
```diff
@@ -0,0 +1,126 @@
+from sys import platform
+
+from PySide2 import QtGui, QtCore
+from PySide2.QtWidgets import (QWidget, QTableWidget, QSizePolicy,
+                               QPushButton, QTableWidgetItem, QVBoxLayout)
+
+c_key = u"\u2318" if platform == 'darwin' else "Ctrl"
```
[the remark this answers is not in the document]
Comment (pshriwise) [reply overlays.py]: Nice! I hadn't even considered pasting the character in. Much more clear.

File: overlays.py [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+            for j in range(n_cols):
+                self.tableWidget.setItem(i, j, ShortcutTableItem())
+
+        self.tableWidget.setShowGrid(False)
+        self.tableWidget.setSizePolicy(QSizePolicy.Expanding,
+                                       QSizePolicy.Expanding)
+        self.tableWidget.verticalHeader().setVisible(False)
+        self.tableWidget.horizontalHeader().setVisible(False)
+        self.tableWidget.setStyleSheet("background-color:" \
+                                       "rgba(30, 30, 30, 230);" \
```
Comment (paulromano) [inline overlays.py]: What I meant was:
```Python
        self.tableWidget.setStyleSheet("background-color: rgba(30, 30, 30, 230);"
                                       "border: 0px;"
                                       "padding: 20px")
```

Comment (pshriwise) [reply overlays.py]: Ah, sorry! Meant to clarify and ended up pushing that change instead. Updating...

**Pull request merged**